automation/test.sh

```diff
@@ -239,7 +239,9 @@ export NAMESPACE="${NAMESPACE:-kubevirt}"
 # Make sure that the VM is properly shut down on exit
 trap '{ make cluster-down; }' EXIT SIGINT SIGTERM SIGSTOP
 
-make cluster-down
+if [ "$CI" != "true" ]; then
+  make cluster-down
+fi
```

Comment:
Could you please elaborate why is it required on a non-CI environment, but not on the CI?

Reply:
Note that we don't use this file locally usually,
on non CI it might be needed only because you might have a cluster already before
see PR desc and commit message it has some info, if more is missing please let me know

Reply:
Thank you for the explanation.
I suggest to add it to the commit message.